trailblazer/dto/create_analysis_request.py

```diff
@@ -10,7 +10,7 @@ class CreateAnalysisRequest(BaseModel):
     out_dir: str
     order_id: int | None = None
     priority: TrailblazerPriority
-    data_analysis: str | None = None
+    workflow: str | None = None
```

Comment:
It is important that this is changed in the trailblazer client in cg as well

Reply:
https://github.com/Clinical-Genomics/cg/pull/2932#